Update bookings-overview.md (#3081)
MicrosoftDocs/microsoft-365-docs: microsoft-365/bookings/bookings-overview.md

- A business-facing mobile app where Bookings calendar owners and administrators can see all of their appointments, access customer lists and contact information, and make manual bookings on the go.

> ## Connect Microsoft Bookings to Facebook Page.

Reviewer: ```suggestion
## Connect Microsoft Bookings to your Facebook page
```

> 1)To remove or connect Microsoft Bookings and Facebook page, open your business Facebook page.

Reviewer: ```suggestion
1. To remove or connect Microsoft Bookings and your Facebook page, open the Facebook page of your organization.
```

> 2)Select **Settings** from the upper right corner and chose **Partner Apps and Services** option from the left panel. 

Reviewer: ```suggestion
2. Select **Settings** from the upper right corner and chose the **Partner Apps and Services** option from the left panel. 
```

> 3)Next hit the **Add Service** button, select **Microsoft Bookings** and click **Add Service** option.

Reviewer: ```suggestion
3. Next, select the **Add Service** button, choose **Microsoft Bookings**, and select the **Add Service** option.
```

> 4)Instantly, a new tab will open. When it is visible, sign into your Office 365 account and find your booking calendars. Here, select the calendar you want to connect to your Facebook page and choose the Connect option.

Reviewer: ```suggestion
4. A new tab will open instantly. When it is visible, sign in to your Office 365 account and find your Bookings calendars. Select the calendar you want to connect to your Facebook page and choose the **Connect** option.
```

> 5)Once you complete the above step, the tab will close, and you’ll return to Facebook. Now you’re all set to receive Bookings through your Facebook page.

Reviewer: ```suggestion
5. Once you've completed the previous step, the tab will close and you’ll be returned to Facebook. Now you’re all set to receive Bookings through your Facebook page.
```

> 6)If you would like to remove Microsoft Bookings and Facebook page, simply select Remove Service on the Page Action button in the Microsoft Bookings tab.

Reviewer: ```suggestion
6. If you would like to remove Microsoft Bookings from your Facebook page, simply select **Remove Service** on the **Page Action** button in the Microsoft Bookings tab.
```
